sycl/test/basic_tests/min_max_test.cpp

```diff
@@ -0,0 +1,12 @@
+// REQUIRES: windows
+// RUN: %clangxx -fsycl %s -o %t.out
```

Comment:
Probably it's enough to just check that there is not diagnostics emitted
similarly to how it's done in ./test/regression/sub-group-store-const-ref.cpp